ydb/library/yql/dq/runtime/dq_output_channel.cpp

```diff
@@ -89,10 +89,6 @@ class TDqOutputChannel : public IDqOutputChannel {
     }
 
     void DoPush(NUdf::TUnboxedValue* values, ui32 width) {
-        ui64 rowsInMemory = PackedRowCount + ChunkRowCount;
-
-        LOG("Push request, rows in memory: " << rowsInMemory << ", bytesInMemory: " << (PackedDataSize + Packer.PackedSizeEstimate())
```

Comment:
Removed these logs because logging each push leads to huge output (1gb per minute)